util/moaiutil.lua

```diff
@@ -5,11 +5,73 @@ require ( 'http' )
 -- setup
 --==============================================================
 
-INVOKE_DIR		= MOAIFileSystem.getAbsoluteDirectoryPath ( arg [ 1 ])
-MOAI_SDK_HOME	= MOAIFileSystem.getAbsoluteDirectoryPath ( arg [ 2 ])
-MOAI_CMD		= arg [ 3 ]
-SCRIPT_DIR		= string.format ( '%sutil/%s/', MOAI_SDK_HOME, MOAI_CMD )
+INVOKE_DIR      = MOAIFileSystem.getAbsoluteDirectoryPath ( arg [ 1 ])
+MOAI_SDK_HOME   = MOAIFileSystem.getAbsoluteDirectoryPath ( arg [ 2 ])
+MOAI_CMD        = arg [ 3 ]
+SCRIPT_DIR      = string.format ( '%sutil/%s/', MOAI_SDK_HOME, MOAI_CMD )
+
+function usage()
+  print([[
+    MOAI Utility Toolbelt 
+
+    PREPARE to use the toolbelt:
+        MOAI SDK has a toolbox environment script (env.sh) for use prior to loading the toolbelt:
+            source $MOAI_SDK_HOME/env.sh
```

Comment:
env.sh works on Linux and OSX of course, and the purpose to have it set up is so that new users have a single script they can run to just up and running.  MOAI_SDK_HOME is used elsewhere in the system too, not just moaiutil ..